flexmeasures/cli/data_add.py

```diff
@@ -1362,7 +1395,7 @@ def add_schedule_for_storage(  # noqa C901
         },
     )
 
-    quantity_or_sensor_vars = {
+    flex_model_quantity_or_sensor_vars = {
```

Comment:
Perhaps, this could be a two level dictionary:

```javascript
quantity_or_sensor_vars = {
    "flex_model" : {
        ...
    },
    "flex_context" : {
        ...
    }
}
```

and then, we can just use 1 two level for loop:

```python
for key in ["flex_model", "flex_context"]:
    for field_name, value in quantity_or_sensor_vars[key].items():
        ...
```